### PR TITLE
Merge 2017-07-10 d24a2734a7

### PR DESCRIPTION
**Pull request by iMichka**
+ [ ] ansible-cmdb
+ [ ] joe
+ [ ] wxmac

### CODE REVIEW

File: Formula/duti.rb [hunk 1]
```diff
@@ -17,11 +17,10 @@ class Duti < Formula
   depends_on "autoconf" => :build
```
Comment (rwhogg) [inline Formula/duti.rb]: We should make this `depends_on :macos`. It fails on install with `checking which SDK to use... configure: error: linux-gnu is not a supported system`.

It's not useful on Linux anyhow.

Comment (sjackman) [inline Formula/duti.rb]: I've opened a PR at https://github.com/Linuxbrew/homebrew-core/pull/3231

**Comment by sjackman** [comment]
```
brew install wxmac
Your build has exceeded the memory limit of 4G on 1 container.
```
https://6671-56942570-gh.circle-artifacts.com/0/tmp/memory-usage.txt
Try 16 jobs.

**Comment by sjackman** [comment]
Superseded by https://github.com/Linuxbrew/homebrew-core/pull/3232

**Pull request closed without merging**